Hummingbird.docc/Tutorials/MongoKitten/MongoKitten-1-Setup.tutorial

```diff
@@ -0,0 +1,139 @@
+                @Code(name: "Create Swift Package", file: mongokitten-setup-02.sh)
+            }
+            @Step {
+                Add the `MongoKitten` dependency.
+                
+                MongoKitten doesn't need any special treatment. It works seamlessly with Hummingbird.
+                @Code(name: "Package.swift", file: mongokitten-setup-04.swift, previousFile: mongokitten-setup-03.swift)
+            }
+        }
+    }
+    
+    @Section(title: "Setup MongoDB") {
+        @ContentAndMedia {
+            MongoDB is a serivce that needs to run alongside your application. It's a prerequisite for running MongoKitten.
+            
+            @Image(source: "mongokitten.png", alt: "MongoKitten logo")
+        }
+        
+        @Steps {
+            @Step {
+                Install and Run [Docker](https://www.docker.com/products/docker-desktop/) if it's not running already.
+                
+                Then, run this command to download and run MongoDB.
+                
+                @Code(name: "Terminal.app", file: install-mongodb-docker.sh)
+            }
+        }
+    }
+    
+    @Section(title: "Add MongoKitten") {
+        @ContentAndMedia {
+            With your Package.swift and database set up, lets add MongoKitten to your project.
+            
+            @Image(source: "mongokitten.png", alt: "MongoKitten logo")
+        }
+        
+        @Steps {
+            @Step {
+                Open `Sources/App/Application+build.swift`.
+                
+                Add the MongoKitten dependency, and modify the `AppArguments` to contain two new variables.
+                @Code(name: "Sources/App/Application+build.swift", file: mongokitten-setup-06.swift, previousFile: mongokitten-setup-05.swift)
+            }
+            
+            @Step {
+                Open `Sources/App/App.swift`
+                
+                This contains an `App` type conforming to `AsyncParsableCommand` with three options, the `hostname` and `port` are used to define the server bind address, `logLevel` sets the level of logging required. Finally the `run()` function which calls `buildApplication(_:)` to create an `Application` and then runs it using `runService()`. You can find out more about the argument parser library [here](https://apple.github.io/swift-argument-parser/documentation/argumentparser).
+                @Code(name: "Sources/App/App.swift", file: mongokitten-setup-07.swift)
+            }
+            @Step {
+                Add the new app arguments with default values.
+                
+                @Code(name: "Sources/App/App.swift", file: mongokitten-setup-08.swift)
+            }
+            
+            @Step {
+                Open `Sources/App/Application+build.swift` again. We can now start the MongoKitten driver.
+                
+                This will connect and login to the server immediately. If the any network hiccups occur, MongoKitten reconnects automatically.
```

Comment:
```suggestion
                This will connect and login to the server immediately. If any network hiccups occur, MongoKitten reconnects automatically.
```